### PR TITLE
doc: fast_pair: Update the documentation

### PR DESCRIPTION
**Pull request by alstrzebonski**
Updates the Fast Pair documentation to reflect the current progress.

Jira: NCSDK-19617

### CODE REVIEW

**Review by mkapala-nordic — APPROVED** [review]

File: doc/nrf/external_comp/bt_fast_pair.rst [hunk 1]
```diff
@@ -13,10 +13,10 @@ For detailed information about supported functionalities, see the official `Fast
 
 .. note::
    The Fast Pair support in the |NCS| is :ref:`experimental <software_maturity>`.
-   The implementation is not yet ready for production and extensions are not supported.
+   The implementation is not yet ready for production and extensions are not fully supported.
 
-   The implementation does not pass end-to-end integration tests in the Fast Pair Validator.
-   The procedure triggered in Android settings is successful (tested with Android 10).
+   The implementation passes Fast Pair Validator tests.
```
Comment (MarekPieta) [inline doc/nrf/external_comp/bt_fast_pair.rst:18]: I think you should mention that you used beta version of the Validator (non-beta would not work)

**Review by divipillai — APPROVED** [review]

**Review by MarekPieta — APPROVED** [review]

**Review by MarekPieta — CHANGES_REQUESTED** [review]
Please inform about the beta version of Fast Pair Validator

**Comment by alstrzebonski** [comment]
One slight change - please rereview.

**Review by MarekPieta — APPROVED** [review]

File: doc/nrf/external_comp/bt_fast_pair.rst [hunk 2]
```diff
@@ -13,10 +13,10 @@ For detailed information about supported functionalities, see the official `Fast
 
 .. note::
    The Fast Pair support in the |NCS| is :ref:`experimental <software_maturity>`.
-   The implementation is not yet ready for production and extensions are not supported.
+   The implementation is not yet ready for production and extensions are not fully supported.
 
-   The implementation does not pass end-to-end integration tests in the Fast Pair Validator.
-   The procedure triggered in Android settings is successful (tested with Android 10).
+   The implementation passes Fast Pair Validator (Beta version) tests.
```
Comment (divipillai) [inline doc/nrf/external_comp/bt_fast_pair.rst:18]:
```suggestion
   The implementation passes Fast Pair Validator (beta version) tests.
```

**Review by divipillai — APPROVED** [review]

**Pull request merged**